homeassistant/helpers/config_validation.py

```diff
@@ -297,6 +297,13 @@ def time(value):
     return time_val
 
 
+def exact_time(value):
+    """Validate timestamp."""
+    if isinstance(value, date) or isinstance(value, datetime):
```

Comment:
For example if there's no quotes on value: 
```
banned_at: 2016-11-23T18:50:33
```

Could be reason of manual adding